index.bs

```diff
@@ -5692,8 +5693,12 @@ interface VTTCue : TextTrackCue {
  <li><p>Let |cue|'s <a>text track cue start time</a> be the value of the |startTime| argument,
  interpreted as a time in seconds.</p></li>
 
- <li><p>Let |cue|'s <a>text track cue end time</a> be the value of the |endTime| argument,
- interpreted as a time in seconds.</p></li>
+ <li><p>If the value of the |endTime| argument is negative Infinity or a Not-a-Number (NaN) value,
+ then throw a <a
+ href="https://tc39.es/ecma262/#sec-native-error-types-used-in-this-standard-typeerror">TypeError
+ </a> exception. Otherwise, let |cue|'s <a>text track cue end time</a> be the value of the |endTime|
+ argument, interpreted as a time in seconds or positive Infinity for an <a>unbounded text track
```

Comment:
Like in HTML, I don't think it's necessary to say "interpreted as a time in seconds or positive Infinity", instead just set the value.

Reply:
I agree. I was trying to match the existing wording for `startTime`, but it's unnecessary. Fixed.